display.js

```diff
@@ -59,14 +59,17 @@ const portfolioData = [
     linkToLiveVersion: '#',
     linkToSource: '#',
   },
+
 ];
 
 const modalContainer = document.createElement('div');
-console.log(modalContainer);
+modalContainer.className = 'popup';
```

Comment:
- [ ] The changes you have introduced have caused the modal popup to stop showing when the `See Project` button is clicked. Kindly fix this.

#### Kindly note that since I am unable to see the popup, kindly ensure  you make the requested changes by the previous reviewer concerning the design and layout